modules/delivery/delivery.service.js

```diff
@@ -0,0 +1,244 @@
+const request = require('request');
+const util = require('util')
+const post = util.promisify(request.post)
+const { 
+    CITY_NOT_FOUND,
+    WAREHOUSE_NOT_FOUND,
+ } = require('../../error-messages/delivery.message')
+
+class NovaPoshtaService {
+  async getNovaPoshtaCities(city){  
+      const res = await post(
```

Comment:
In this request, we duplicate the URL, some elements of the body, and `apiKey`, the only thing that changes is `body` property, so it is better to create a basic method that will take an object with method definition, arguments etc as an argument

Reply:
fixed